app/app.py

```diff
@@ -216,6 +220,70 @@ def initialize_schools(
                     )
 
 
+async def run_factories(db: AsyncSession, settings: Settings) -> None:
+    """Run the factories to create default data in the database"""
+    hyperion_error_logger = logging.getLogger("hyperion.error")
+    if not settings.USE_FACTORIES:
+        hyperion_error_logger.info("Startup: Factories are disabled")
```

Comment:
As this feature is not expected to be run in production, I think we don't need to have a log each time the prod server starts.

I would suggest logging a message when factories are enabled and run instead